nuxeo-document-preview.html

```diff
@@ -232,10 +232,11 @@
               // Insert our previewer
               delete previewer.__templatizeOwner;
               this.templatize(previewer);
-              const instance = this.stamp();
+              let instance = this.stamp();
```

Comment:
this is a local variable so I'm guessing it won't really hold the reference to the instance...

Reply:
yes, doesn't matter and was pushed by mistake 👍 ✅